src/components/sbb-menu/sbb-menu.scss

```diff
@@ -0,0 +1,163 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-menu-position-x: 0;
+  --sbb-menu-position-y: 0;
+  --sbb-menu-min-height: #{px-to-rem-build(120)};
+  --sbb-menu-max-width: #{px-to-rem-build(320)};
+  --sbb-menu-min-width: #{px-to-rem-build(240)};
+}
+
+// stylelint-disable plugin/stylelint-bem-namics
+::slotted(:not(sbb-menu-action)) {
+  display: block;
+  padding-inline: var(--sbb-spacing-fixed-5x);
+}
+// stylelint-enable plugin/stylelint-bem-namics
+
+::slotted(sbb-divider) {
+  --divider-color: var(--sbb-color-iron-default);
+
+  margin: var(--sbb-spacing-fixed-2x) 0;
+}
+
+.sbb-menu {
+  @include scrollbar('dark');
+
+  --sbb-menu-transform: translateY(100%);
+
+  max-height: calc(100vh - var(--sbb-spacing-fixed-8x));
+  min-width: 100%;
+  text-align: start;
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  top: unset;
+  right: unset;
+  color: var(--sbb-color-white-default);
+  border: none;
+  border-radius: var(--sbb-border-radius-4x) var(--sbb-border-radius-4x) 0 0;
+  background-color: var(--sbb-color-black-default);
+  padding-inline: 0;
+  padding-block: var(--sbb-spacing-fixed-1x);
+
+  &[open] {
+    animation: {
+      name: show;
+      duration: var(--sbb-animation-duration-6x);
+      timing-function: ease;
+    }
+
+    &.sbb-menu--no-animation {
+      animation-duration: 0s;
+    }
+  }
+
+  &::backdrop {
```

Comment:
On the `::backdrop` it is impossible to use any design tokens, so inline values must be used (for background and animation duration in this case).

To learn more: [https://stackoverflow.com/questions/58818299/css-variables-not-working-in-dialogbackdrop](url)

Reply:
Good find 👍 
Please add this as documentation to the above line